src/Integration/Service/SonarQubeServiceWrapper.cs

```diff
@@ -318,6 +326,11 @@ public Uri CreateProjectDashboardUrl(ConnectionInformation connectionInformation
 
         #region Helpers
 
+        internal /*for testing purposes*/ static string GetServerLanguageKey(Language language)
+        {
+            return languageKeys[language];
```

Comment:
I'll add an assert. If we're given a language we don't know about or support then there's nothing we can do here.